### PR TITLE
build: update node 20

### PR DESCRIPTION
**Pull request by joannaskao**

### CODE REVIEW

**Review by kavanagh — COMMENTED** [review]
Thanks for taking a look at this. Just one comment...

File: package.json [hunk 1]
```diff
@@ -49,9 +49,11 @@
...
   },
   "volta": {
-    "node": "16.15.1"
+    "node": "20.11.1",
+    "npm": "10.2.4"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=20",
+    "npm": ">10"
```
Comment (kavanagh) [inline package.json:57]: Is there a reason the npm version needs to specified? I'd prefer if it wasn't defined so we let it be whatever is shipped with the version of Node.

File: package.json [hunk 2]
```diff
@@ -49,9 +49,11 @@
     "semantic-release": "^19.0.2"
   },
   "volta": {
-    "node": "16.15.1"
+    "node": "20.11.1",
+    "npm": "10.2.4"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=20",
```
Comment (kavanagh) [inline package.json:56]: Hmm thinking about it, this should probably be `>=18` because Bertha is on 18 at the moment.

File: package.json (hunk 1 above)
Comment (joannaskao) [reply package.json:57]: Oh yeah, we can get rid of the `npm` bit.

File: package.json (hunk 2 above)
Comment (joannaskao) [reply package.json:56]: Ah... okay thought I'd try to skip straight to 20 so we don't have to do this as often, but that makes sense

**Pull request merged**